src/menu/menu_priv.c

```diff
@@ -0,0 +1,104 @@
+/**
+ * @file    menu_priv.c
```

Comment:
Could rename file to menu_config.c

We could also keep an arrangement where all such structures(evt_cfg/flow_step) are defined in this file.

Reply:
Updated the file name, yes we could definitely use it to store similar configurations